src/ipc/sync-usb-drive.ts

```diff
@@ -1,10 +1,10 @@
 import { IpcMain, IpcMainInvokeEvent } from 'electron';
-import promisifiedExec from '../utils/promisifiedExec';
+import exec from '../utils/exec';
 
 export const channel = 'syncUsbDrive';
 
 async function syncUsbDrive(mountPoint: string): Promise<void> {
-  await promisifiedExec(`sync -f ${mountPoint}`);
+  await exec(`sync -f ${mountPoint}`);
```

Comment:
To fully make this safe, you need to use the broken out approach in calling `exec`:

`exec("sync", ["-f", mountPoint]);`

A good rule is, if you're using string interpolation in the first parameter to `exec`, you've almost certainly got a security issue.

Reply:
Oops! I understand the rule and remembered to switch it on the other PR, but blanked here. Thank you. Fixed and squashed.